charts/data-dashboard-backend/values.yaml

```diff
@@ -7,9 +7,11 @@ replicaCount: 1
 
 image:
   # -- docker image repository
-  repository: ghcr.io/thehyve/radar-data-dashboard-backend
+  repository: radarbase/radar-data-dashboard-backend
   # -- image pull policy
   pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
```

Comment:
The comments should start with a `--` in order to be added into README.md

Reply:
I did not know. Fixed!

Reply:
It doesn't seem to be fixed